server/app/IcepeakTokenGen/Main.hs

```diff
@@ -13,7 +13,7 @@ import           AccessControl
 import           JwtAuth
 
 data Config = Config
-  { configJwtSecret      :: Maybe JWT.Signer
+  { configJwtSecret      :: Maybe JWT.EncodeSigner
```

Comment:
Maybe it would be a good idea to include a comment for this field so it's clear what it means if an `EncodeSigner` is not provided (similar to the comment already in `server/src/Config.hs`